pom.xml

```
@@ -187,6 +187,11 @@ Copyright (c) 2012 - Jeremy Long
         <defaultGoal>clean install</defaultGoal>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>com.googlecode.maven-download-plugin</groupId>
+                    <artifactId>download-maven-plugin</artifactId>
+                    <version>1.13.0</version>
```

```suggestion
                    <groupId>io.github.download-maven-plugin</groupId>
                    <artifactId>download-maven-plugin</artifactId>
                    <version>2.0.0</version>
```

Artifact was relocated; but no other changes it seems. https://github.com/download-maven-plugin/download-maven-plugin/releases/tag/2.0.0